ADMVEDU95/ADMVEDU95/Presentation/Category/View/CategoryViewController.swift

```diff
@@ -38,26 +25,31 @@ class CategoryViewController: UIViewController {
     private func setupTableView() {
         tableView.delegate = self
         tableView.dataSource = self
-        tableView.register(UITableViewCell.self, forCellReuseIdentifier: Constants.categoryCellIdentifier)
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: UITableViewCell.reuseIdentifier)
```

Comment:
Is a default cell used instead of custom one? 
Should it be registered in this case?
Was CategoryTableViewCell a default cell as well?